streampipes-extensions/streampipes-sinks-notifications-jvm/src/main/java/org/apache/streampipes/sinks/notifications/jvm/slack/SlackNotificationSink.java

```diff
@@ -56,10 +55,9 @@ public class SlackNotificationSink extends StreamPipesDataSink {
   private String originalMessage;
 
   @Override
-  public DataSinkDescription declareModel() {
-
+  public DataSinkBuilder declareModelWithoutSilentPeriod() {
     return DataSinkBuilder
-        .create("org.apache.streampipes.sinks.notifications.jvm.slack", 0)
+        .create("org.apache.streampipes.sinks.notifications.jvm.slack", 1)
```

Comment:
Does it make sense to extract the ID to a public static variable so that it can be reused in the Migration classes?